deploy/charts/firefly/README.md

```diff
@@ -194,15 +194,46 @@ core:
     - name: FIREFLY_PUBLICSTORAGE_IPFS_API_AUTH_USERNAME
       valueFrom:
         secretKeyRef:
-          name: my-ipfs-basic-auth
+          name: custom-ipfs-basic-auth
           key: username
     - name: FIREFLY_PUBLICSTORAGE_IPFS_API_AUTH_PASSWORD
       valueFrom:
         secretKeyRef:
-          name: my-ipfs-basic-auth
+          name: custom-ipfs-basic-auth
           key: password
 ```
 
+In the case of PostgreSQL credentials, environment variables will have to be set for FireFly and its [migrations `Job`](#database-migrations):
+
+```yaml
+# database section of the config still needs to be set to indicate Postgres
```

Comment:
Any significance of "still" in the sentence here?

Reply:
Ah nope, will remove!